### PR TITLE
DOCAPI-7438: Update of quantileTiming docs.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #5911

### CODE REVIEW

File: docs/en/query_language/agg_functions/reference.md [hunk 1]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
+**Accuracy**
+
+The calculation is accurate if:
+
+- Total number of values is not more than about 5670.
+- Total number of values is more than about 5670, but the times of page loading is less than 1024ms.
+
+Otherwise, the result of a calculation is rounded to the value, multiple of 16 ms.
+
+For calculating quantiles of page loading times, using this function is more effective, and the result is more accurate than for the [quantile](#agg_function-quantile) function.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]:
```suggestion
!! note "Note"
    For calculating quantiles of page loading times, this function is more effective and accurate compared to [quantile](#agg_function-quantile).
```

File: docs/en/query_language/agg_functions/reference.md [hunk 2]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
+
+**Parameters**
 
-If the total value is not more than about 5670, then the calculation is accurate.
+- `level` — Quantile level. Range: [0, 1].
+- `expr` — [Expression](../syntax.md#syntax-expressions) returning number in the [Float*](../../data_types/float.md) type.
+    
+    - When passing negative values to the function, the behavior is undefined.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]:
```suggestion
    - If negative values are passed to the function, the behavior is undefined.
```

File: docs/en/query_language/agg_functions/reference.md [hunk 3]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
+The calculation is accurate if:
+
+- Total number of values is not more than about 5670.
+- Total number of values is more than about 5670, but the times of page loading is less than 1024ms.
+
+Otherwise, the result of a calculation is rounded to the value, multiple of 16 ms.
+
+For calculating quantiles of page loading times, using this function is more effective, and the result is more accurate than for the [quantile](#agg_function-quantile) function.
+
+**Parameters**
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: I'd swap Parameters and Accuracy

File: docs/en/query_language/agg_functions/reference.md [hunk 4]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
 
-If the total value is not more than about 5670, then the calculation is accurate.
+- `level` — Quantile level. Range: [0, 1].
+- `expr` — [Expression](../syntax.md#syntax-expressions) returning number in the [Float*](../../data_types/float.md) type.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: we'd better explicitly mention that it's supposed to be in milliseconds (btw why it's float then?)

File: docs/en/query_language/agg_functions/reference.md [hunk 5]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
 
-- if the time is less than 1024 ms, then the calculation is accurate.
-- otherwise the calculation is rounded to a multiple of 16 ms.
+- Quantile of the specified level.
 
-When passing negative values to the function, the behavior is undefined.
+Type: `Float32`.
 
-The returned value has the Float32 type. If no values were passed to the function (when using `quantileTimingIf`), 'nan' is returned. The purpose of this is to differentiate these instances from zeros. See the note on sorting NaNs in "ORDER BY clause".
+If no values were passed to the function (when using `quantileTimingIf`), [NaN](../../data_types/float.md#data_type-float-nan-inf) is returned. The purpose of this is to differentiate these cases from the cases which result in zero. See [ORDER BY clause](../select.md#select-order-by) for the note on sorting `NaN` values.
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]: this deservers note/warning block

File: docs/en/query_language/agg_functions/reference.md [hunk 6]
```diff
@@ -846,27 +846,53 @@ To achieve this, the function takes a second argument – the "determinator". Th
...
 
 The result is determinate (it doesn't depend on the order of query processing).
```
Comment (blinkov) [inline docs/en/query_language/agg_functions/reference.md]:
```suggestion
The result is deterministic (it doesn't depend on the order of query processing).
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/query_language/agg_functions/reference.md (hunk 4 above)
Comment (BayoNet) [reply docs/en/query_language/agg_functions/reference.md]: From the dev pull: "Returned back support for floating point argument in function quantileTiming for backward compatibility."

**Pull request merged**